src/Hedgehog/Property.fs

```diff
@@ -418,15 +418,17 @@ module Property =
     let recheckBool (size : Size) (seed : Seed) (g : Property<bool>) : unit =
         bind g ofBool |> recheck size seed
 
+    let asString' (n : int<tests>) (p : Property<unit>) : string =
+        Report.render <| report' n p
```

Comment:
Do we really want to keep using `<|` in our code? https://youtu.be/1AZA1zoP-II?t=2130

Reply:
There are plenty of other spots in the code where they are used, so mostly sticking to convention. I agree though that we should probably avoid it, and use `x |> f` or `(f x)` instead. Should I tackle that in this PR or put up another quick one after this?

Reply:
I've created #276 for this.